backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/VmCommand.java

```diff
@@ -643,8 +639,8 @@ public boolean isVmDuringBackup() {
                 .anyMatch(vmBackup -> !vmBackup.getPhase().isBackupFinished());
     }
 
-    private boolean shouldBlockDuringBackup() {
-        return getParentParameters() == null || getParentParameters().getCommandType() != ActionType.HybridBackup;
+    protected boolean shouldBlockDuringBackup() {
```

Comment:
should also be overridden by hibernate and reboot commands, right?

Reply:
ack, unblocked VmOperationCommandBase